README.md

```diff
@@ -24,6 +24,8 @@ addCompilerPlugin("ch.epfl.scala" %% "scalac-profiling" % "<version>" cross Cros
 
 To learn how to use the plugin, read [Speeding Up Compilation Time with `scalac-profiling`](https://www.scala-lang.org/blog/2018/06/04/scalac-profiling.html) in the scala-lang blog.
 
+Note that in Scala 2.13, the preferred form of the compiler option to enable statistics is `-Vstatistics`. It is part of the family of `-V` flags that enable various "verbose" behaviors. (In 2.12, the flag is called `-Ystatistics`.)
+
```

Comment:
In fact, without having `-Vstatistics`/`-Ystatistics` enabled, the compiler plugin won't produce any data, so given that, I'd put this into the `Install` section as one of the crucial prerequisites.

Reply:
Agree, that sounds like a good improvement. (I hit "merge" on this version regardless.)